web/yaamp/core/trading/livecoin_trading.php

```diff
@@ -228,7 +228,7 @@ function doLiveCoinTrading($quick = false)
 			sleep(1);
 
 			$res = $livecoin->sellLimit($pair, $sellprice, $balance);
-			if (!$res->success == 'true' && $res->added == 'true')) {
+			if (!$res->success == 'true' && $res->added == 'true') {
```

Comment:
are you sure its a string ?

https://www.livecoin.net/api/orders#exchangeselllimit